src/config.rs

```diff
@@ -1095,6 +1132,10 @@ impl Default for Font {
             offset: FontOffset {
                 x: 0.0,
                 y: 0.0
+            },
+            glyph_offset: GlyphOffset {
```

Comment:
It would be a lot cleaner to `#[derive(Default)]` for `GlyphOffset` and use `Default::default()` here.